javaparser-core/src/main/java/com/github/javaparser/ast/expr/ObjectCreationExpr.java

```diff
@@ -44,9 +44,12 @@
 import com.github.javaparser.TokenRange;
 
 /**
- * A constructor call. <br/>In <code>new HashMap.Entry<String, Long>(15) {public String getKey() {return null;}};</code>
- * HashMap is the scope, Entry is the type, String and Long are type arguments, 15 is an argument, and everything in { }
+ * A constructor call.
+ * <br/>In <code>new HashMap.Entry&lt;String, Long>(15) {public String getKey() {return null;}};</code>
```

Comment:
Can be, but HTML is fine with this. Keeps it a little more readable for people reading it from their IDE.